policyengine_canada/parameters/gov/cra/tax/income/credits/age_amount_credit/amount/reduction.yaml

```diff
@@ -0,0 +1,32 @@
+description: Canada Age Amount Credit higher credit amount.
+brackets:
+  - threshold:
+      values:
+        2021-01-01: 0
+    rate:
+      values:
+        2021-01-01: 0
+  - threshold:
+      values:
+        2021-01-01: 38_893
+    rate:
+      values:
+        2021-01-01: 0.15
+  - threshold:
```

Comment:
$90,313 is mechanically derived from $38,893 and the credit amount

$38,893 + $7,713/15% = $90,313

So you can remove the $90,313 from both here and the income cap, and then ensure nonnegativity in the Python